app/brave_generated_resources.grd

```diff
@@ -403,7 +403,7 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
         Show the number of blocked items on the Shields icon
       </message>
       <message name="IDS_SETTINGS_BRAVE_SHIELDS_GOOGLE_LOGIN_LABEL" desc="Label for a switch control which allows Google social buttons to be enabled/disabled">
-        Allow Google login buttons on third party sites
+        Allow Google login in extensions and third party sites
```

Comment:
suggest adding a link to a page (wiki?) explaining what google login in extensions entails

Reply:
@diracdeltas - This is the wiki: https://developer.chrome.com/apps/identity - I can add it as a comment in the code. Or do you want me to create a separate brave wiki for it?

Reply:
@jumde i mean add a page that explains to the user what this option means and what data gets sent to Google. i think most people understand what Google login in 3p sites means but it's not clear what gets sent to google when you enable google login in extensionos.

Reply:
@diracdeltas - Here it is: https://github.com/brave/brave-browser/wiki/Social-Blocking:-Allow-Google-login

Reply:
wiki page looks good, can you add an info link to it next to the setting?

Reply:
@diracdeltas - Set here: https://github.com/brave/brave-core/pull/5616/files#diff-2ff48ebe156c663642e6a503ed3d3913R17